ch06.adoc

```diff
@@ -80,7 +80,46 @@ data:
 ====
 
 
+[[taxon-names-and-identifiers, Section 6.1.2. "Taxon Names and Identifiers"]]
+==== Taxon Names and Identifiers
 
+A taxon is a named level within a biological classification, such as a class, genus and species. Quantities dependent on taxa have generic standard names containing the phrase "organisms_in_taxon", and the taxa are identified by auxiliary coordinate variables.
+
+The taxon auxiliary coordinate variables are string-valued. The plain-language name of the taxon must be contained in a variable with **`standard_name`** of **`biological_taxon_name`**. A Life Science Identifier (LSID) may be contained in a variable with **`standard_name`** of **`biological_taxon_lsid`**. This is a URN with the syntax "urn:lsid:<Authority>:<Namespace>:<ObjectID>[:<Version>]". This includes the reference classification in the <Authority> element and these are restricted by the LSID governance. It is strongly recommended in CF that the authority chosen is World Register of Marine Species (WoRMS) for oceanographic data and Integrated Taxonomic Information System (ITIS) for freshwater and terrestrial data. WoRMS LSIDs are built from the WoRMS AphiaID  taxon identifier such as "urn:lsid:marinespecies.org:taxname:104464" for AphiaID 104464. This may be converted to a URL by adding prefixes such as ​http://lsid.twg.org/. ITIS LSIDs are built from the ITIS Taxonomic Serial Number (TSN), such as "urn:lsid:itis.gov:itis_tsn:180543".
```

Comment:
http://lsid.twg.org/ should be changed to http://www.lsid.info
```suggestion
The taxon auxiliary coordinate variables are string-valued. The plain-language name of the taxon must be contained in a variable with **`standard_name`** of **`biological_taxon_name`**. A Life Science Identifier (LSID) may be contained in a variable with **`standard_name`** of **`biological_taxon_lsid`**. This is a URN with the syntax "urn:lsid:<Authority>:<Namespace>:<ObjectID>[:<Version>]". This includes the reference classification in the <Authority> element and these are restricted by the LSID governance. It is strongly recommended in CF that the authority chosen is World Register of Marine Species (WoRMS) for oceanographic data and Integrated Taxonomic Information System (ITIS) for freshwater and terrestrial data. WoRMS LSIDs are built from the WoRMS AphiaID  taxon identifier such as "urn:lsid:marinespecies.org:taxname:104464" for AphiaID 104464. This may be converted to a URL by adding prefixes such as ​http://www.lsid.info/. ITIS LSIDs are built from the ITIS Taxonomic Serial Number (TSN), such as "urn:lsid:itis.gov:itis_tsn:180543".
```